examples/examples.css

```diff
@@ -2,28 +2,28 @@ html {
   background-color: #34405d;
 }
 body {
-  font: message-box;
```

Comment:
This was reorganized by my vim plugin running stylelint.

Reply:
Are we not including this in our linting?

Reply:
Looks like it's not:
```
    "lint-css": "stylelint 'src/**/*.css' 'res/**/*.css'",
```